plugins/modules/docker_network.py

```diff
@@ -300,6 +313,10 @@ def __init__(self, client):
         for key, value in client.module.params.items():
             setattr(self, key, value)
 
+        # config_only sets driver to 'null' so force that here otherwise we get
+        # diffs of 'null' --> 'bridge' given driver defaults to 'bridge'.
```

Comment:
```suggestion
        # config_only sets driver to 'null' (and scope to 'local') so force that here. Otherwise we get
        # diffs of 'null' --> 'bridge' given that the driver option defaults to 'bridge'.
```